.eslintrc

```diff
@@ -14,7 +14,8 @@ module.exports = {
         "no-param-reassign": 0,
         "comma-dangle": 0,
```

Comment:
You may want to limit the max-length of lines to make your code more readable, a max-length of 80 - 100 characters is considered acceptable.